.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: true

```

Comment:
enable blank issue template, user still can create the issue with free format just in case if the default templates do not work for them.